pkg/model/lease.go

```diff
@@ -0,0 +1,89 @@
+package model
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Lease is a type corresponding to a Lease
+// table record
+type Lease struct {
+	AccountID                string                 `json:"AccountId"`                // AWS Account ID
```

Comment:
Missing the dynamodbav tags.  We are going to need those.

Reply:
Added